Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -7857,7 +7857,7 @@ def is_demisto_version_ge(version, build_number=''):
         raise
     except ValueError:
         # dev editions are not comparable
-        demisto.log(
+        demisto.log(  # pylint: disable=E9012
```

Comment:
let's not ignore this one, and change it to demisto.error

Reply:
done

Reply:
Revertig to demisto.log and ignoring as it fails unit tests.